backend/FwLite/LcmCrdt/Changes/SetPartOfSpeechChange.cs

```diff
@@ -12,18 +12,17 @@ public override async ValueTask ApplyChange(Sense entity, ChangeContext context)
         if (PartOfSpeechId is null)
         {
             entity.PartOfSpeechId = null;
-            entity.PartOfSpeech = string.Empty;
             return;
         }
 
         var partOfSpeech = await context.GetCurrent<PartOfSpeech>(PartOfSpeechId.Value);
         if (partOfSpeech is null or { DeletedAt: not null })
         {
             entity.PartOfSpeechId = null;
-            entity.PartOfSpeech = string.Empty;
+            entity.PartOfSpeech = null;
             return;
         }
         entity.PartOfSpeechId = partOfSpeech.Id;
-        entity.PartOfSpeech = partOfSpeech.Name["en"];
+        entity.PartOfSpeech = partOfSpeech;
```

Comment:
this may cause problem, I'm not totally sure though.

Reply:
I *think* this is correct. The tests pass with this in place (after your test fixes from yesterday), but commenting out the `entity.PartOfSpeech = partOfSpeech` line causes LcmCrdt.Tests.MiniLcmTests.BasicApiTests.UpdateSensePartOfSpeech to fail. (Integration tests still have about 20 minutes to run before I see which of them fail).